core/amber/build.sbt

```diff
@@ -35,7 +35,7 @@ PB.generate / excludeFilter := "scalapb.proto"
 
 /////////////////////////////////////////////////////////////////////////////
 // Akka related
-val akkaVersion = "2.6.12"
+val akkaVersion = "2.8.3"
```

Comment:
It's better to use a separate PR to bump dependency version. Maybe do it for future cases.